test/e2e/e2eTests.js

```diff
@@ -96,7 +96,7 @@ casper.test.begin('Back to login page to login', 5, function suite(test) {
         // 'true/false' is whether to submit form
         this.fill('form.login-form', {
             'username': "gsadmin",
-            'password': "solinea"
+            'password': "admin"
```

Comment:
Great idea. Will add variables to top of file and research passing params to Casper.js 



> On Sep 1, 2015, at 8:09 AM, John Stanford <notifications@github.com> wrote:
> 
> In test/e2e/e2eTests.js:
> 
> > @@ -96,7 +96,7 @@ casper.test.begin('Back to login page to login', 5, function suite(test) {
> >          // 'true/false' is whether to submit form
> >          this.fill('form.login-form', {
> >              'username': "gsadmin",
> > -            'password': "solinea"
> > +            'password': "admin"
> For future consideration: can references to usernames/passwords be variables that are loaded from environment or command-line parameters? Minimally, pulled up to the top of the file to allow for easier editing?
> 
> —
> Reply to this email directly or view it on GitHub.
>